emails/payin_bankwire_created.spt

```diff
@@ -2,8 +2,8 @@
 
 [---] text/html
 <p>{{ _(
-    "We are ready to receive the funds. Please send exactly {0} to the "
-    "following account using the reference code below so we'll know the "
+    "Liberapay is ready to receive the funds. Please send exactly {0} to the "
```

Comment:
There is a lot of "we" going on, which is just a barrier in terms of community and clarity.